### PR TITLE
Inactive user placing order issue fixed #6407

### LINKED ISSUE #6407: The user should not be able to place the order if the user is inactive.
**Bagisto Version:** Master

**Steps To reproduce**--

**1.** Log in as a customer
**2.** Add any item to the cat.
**3.** Click on place order.
**4.** Select the payment method.
**5.** log in as admin.
**6.** Inactive the same user.
**7.** Go to the shop as a customer
**8.** continue the order process.

**Expected result:** The user should not be **able** to **place** the **order** if the user is **inactive**.

**Actual result:** The user is able to place the order while inactive. Please find the screen recording [Here](https://www.awesomescreenshot.com/video/8797143?key=042fdabc2c5b1e3437d55c1247990f25)

### PR DESCRIPTION
**Pull request by papnoisanjeev**
## Issue Reference
#6407

## Description
Inactive user should not allow to place an order.

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
The warning message is not correct, please find the screenshot [Here](https://prnt.sc/5ZFag38flP0v)
The message should be like: **Your account is Inactive by admin.**

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
It's working fine now, please find the screenshot [Here](https://prnt.sc/jz6jKpXUwYQk)

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Shop/src/Http/Controllers/OnepageController.php [hunk 1]
```diff
@@ -230,6 +230,10 @@ public function validateOrder()
             throw new \Exception(trans('shop::app.checkout.cart.suspended-account-message'));
         }
 
+        if(auth()->guard('customer')->user() && ! auth()->guard('customer')->user()->status) {
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/OnepageController.php:233]: Add space after `if`.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Shop/src/Http/Controllers/OnepageController.php (hunk 1 above)
Comment (papnoisanjeev) [reply packages/Webkul/Shop/src/Http/Controllers/OnepageController.php]: @devansh-webkul  done.

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**